Packs/ApiModules/Scripts/MicrosoftApiModule/MicrosoftApiModule.py

```diff
@@ -1360,6 +1364,17 @@ def start_auth(self, complete_command: str) -> str:
 and enter the code **{user_code}** to authenticate.
 2. Run the **{complete_command}** command in the War Room."""
 
+    def is_auth_code_reconfigured(self) -> bool:
+        if not self.auth_type == SELF_DEPLOYED_AUTH_TYPE:
+            return False
```

Comment:
self.auth_code is defined only for `self deployed` otherwise auth_code not defined error will be thrown